credentials/alts/internal/handshaker/handshaker_test.go

```diff
@@ -219,6 +220,7 @@ func (s) TestServerHandshake(t *testing.T) {
 			go func() {
 				_, context, err := shs.ServerHandshake(ctx)
 				if err == nil && context == nil {
+					errc <- nil
```

Comment:
Instead of pushing `nil` into `errc`, I think it would be better to push an error, something like `errc <- errrors.New("expected non-nil ALTS context")` and remove the call to `panic`.

Reply:
Good idea! I will replace this line with `errc <- errrors.New("expected non-nil ALTS context")`, and replace `panic()` with `return`